### ISSUE TITLE
Wrapping mapper still fails if there's nothing to wrap ;)

### ISSUE THREAD

**Issue by aflatter** [rom-rb/rom#179]
Thanks for all the help today! I think I stumbled into another issue, the following might suffice to repro:

```ruby
class FooMapper < ROM::Mapper
  relation :foos
  register_as :foo
  inherit_header false

  wrap :test, prefix: :test do
    attribute :id
  end
end
```

```
2.2.1 :006 > ROM.env.mappers.foos.foo.transformer[[{}]]
ArgumentError: odd number of arguments for Hash
	from /vagrant/vendor/transproc/lib/transproc/hash.rb:12:in `[]'
	from /vagrant/vendor/transproc/lib/transproc/hash.rb:12:in `block in <module:Transproc>'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:11:in `[]'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:11:in `call'
	from /vagrant/vendor/transproc/lib/transproc/hash.rb:21:in `block in <module:Transproc>'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:11:in `[]'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:11:in `call'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:16:in `block in compose'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:11:in `[]'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:11:in `call'
	from /vagrant/vendor/transproc/lib/transproc/array.rb:8:in `block (2 levels) in <module:Transproc>'
	from /vagrant/vendor/transproc/lib/transproc/array.rb:8:in `map!'
	from /vagrant/vendor/transproc/lib/transproc/array.rb:8:in `block in <module:Transproc>'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:11:in `[]'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:11:in `call'
	from /vagrant/vendor/transproc/lib/transproc/array.rb:3:in `block in <module:Transproc>'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:11:in `[]'
	from /vagrant/vendor/transproc/lib/transproc/function.rb:11:in `call'
	from (irb):6
```

**Comment by solnic** [comment]
ah, yeah I know why it happens, this makes me think that we do want an empty hash from `nest` instead of nil.

makes sense?

**Comment by aflatter** [comment]
Yup, I hope so. :)

**Comment by solnic** [comment]
OK fixed in transproc 0.1.2
